src/ocsp.c

```diff
@@ -1244,6 +1244,7 @@ int wolfSSL_OCSP_request_add1_nonce(OcspRequest* req, unsigned char* val,
 #ifndef HAVE_FIPS
             wc_InitRng_ex(&rng, req->heap, INVALID_DEVID)
 #else
+WOLFSSL_ABI
```

Comment:
Don't put WOLFSSL_ABI on function calls.

Reply:
Good call. Removed the incorrect WOLFSSL_ABI placing.